src/target/ext/base/multithread.rs

```diff
@@ -110,6 +111,23 @@ pub trait MultiThreadBase: Target {
     ) -> Option<crate::target::ext::thread_extra_info::ThreadExtraInfoOps<'_, Self>> {
         None
     }
+
+    /// Override the reported PID when running in multithread mode (default: 1)
+    ///
+    /// When implementing gdbstub on a platform that supports multiple
+    /// processes, the active PID needs to match the attached process.
+    /// Failing to do so will cause GDB to fail to attach to the target
+    /// process.
+    ///
+    /// This should reflect the currently-debugged process which should be
+    /// updated when switching processes after calling `attach()`.
```

Comment:
can you set up an intra-doc link that points at the specific `attach` method this is referring to? i.e: the one in the ExtendedMode IDET

Reply:
There is now an intra-doc link